src/socket/addrinfo.cr

```diff
@@ -82,12 +82,12 @@ class Socket
     class Error < Socket::Error
       getter error_code : Int32
 
-      def self.new(error_code)
-        new error_code, "getaddrinfo: #{String.new(LibC.gai_strerror(error_code))}"
+      def self.new(error_code, domain)
```

Comment:
```suggestion
      def self.new(error_code, hostname)
```
We're looking up hostnames, not domains.

Reply:
Yeah the getaddrinfo man page seems to refer to it as a "node"  or something like that.  Referring to it as a "domain" is scattered throughout the entire file and comments (plus domain can be an IP address or a hostname), OK to leave as domain for now?  Cheers!

Reply:
Yeah, fine. They should all be changed to hostname or address.